### PR TITLE
Added controls via board for observers

### PR DESCRIPTION
**Pull request by theQuazz**

### CODE REVIEW

File: wurst/systems/boards/ObserverBoard.wurst [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        for observer in observers
+            let options = asList<ObserverBoardControl>(
+                new ObserverBoardVisionControl(observer),
+                new ObserverBoardZoomControl(observer)
+            )
+            new ObserverBoard(observer, options)
+
+        destroy observers
+
+    registerPlayerEvent(EVENT_PLAYER_END_CINEMATIC) ->
```
Comment (jonathanmclaus) [inline wurst/systems/boards/ObserverBoard.wurst]: What is this supposed to trigger on?

Comment (theQuazz) [reply wurst/systems/boards/ObserverBoard.wurst]: `Esc` button

Comment (jonathanmclaus) [reply wurst/systems/boards/ObserverBoard.wurst]: That gets fired even if there isn't a cinematic to end?

Comment (theQuazz) [reply wurst/systems/boards/ObserverBoard.wurst]: Yes

**Pull request merged**